modules/backend/widgets/Form.php

```diff
@@ -338,7 +353,7 @@ public function onRefresh()
                 if (!isset($this->allFields[$field])) {
                     continue;
                 }
-
+                /** @var FormWidgetBase $fieldObject */
```

Comment:
Inline `@var` tags like this allow IDEs to resolve the method used on objects (and therefore also their return types)